tests/core/models/universalresults.js

```diff
@@ -0,0 +1,14 @@
+import coreUniversalResponse from '../../fixtures/core/coreuniversalresponse.json';
+import sdkUniversalResults from '../../fixtures/core/sdkuniversalresults.json';
+import UniversalResults from '../../../src/core/models/universalresults';
+
+it('constructs a universal results model from an answers-core universal results model', () => {
```

Comment:
could we add a comment here and/or in src/.../universalresults that urls is deprecated? We have verticalPages config now, which does the same thing (see here) https://github.com/yext/answers/blob/da4f923fa33a6974392f71106ae8fcf12429b738/src/ui/components/search/searchcomponent.js#L622

Reply:
If it's deprecated, is now a good time to remove it? If not, i'll add a comment in src/.../universalresults.js

Reply:
Yeah I think it would be great to remove it!

Reply:
the "less temporary hack for DEMO"s the better

Reply:
If it's too much we can do it in another PR though

Reply:
ya, let's do this in another PR